person_class.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
```

Comment:
- [ ] Please inherit the `Person` class from the `Nameable` class and create a method `correct_name` that should return the name as requested in the project requirements.
![image](https://user-images.githubusercontent.com/83803414/187511770-91d4853c-5657-4545-b179-4913e05b92df.png)

```ruby
  def correct_name
    name
 end